cli/src/test/java/org/apache/iotdb/tool/integration/ExportCsvTestIT.java

```diff
@@ -63,25 +68,20 @@ protected void testOnWindows() throws IOException {
             "-h",
             "127.0.0.1",
             "-p",
-            "6668",
+            "6667",
             "-u",
             "root",
             "-pw",
             "root",
             "-td",
-            "./");
-    testOutput(builder, output);
+            ".\\",
+            "-q",
+            queryCommand);
+    testOutput(builder, null);
```

Comment:
.....what is the meaning of this test???

You need:
1. write some data into iotdb in setup()
2. export and check whether the result is correct.